requirements.txt

```diff
@@ -126,7 +126,6 @@ graphql-core==2.3.2
     #   graphql-relay
 graphql-relay==2.0.1
     # via graphene
-icalendar==4.0.7
```

Comment:
ahh, this one got removed by pip-compile

Reply:
Looks like it's my bad, it's missing from req.in 🤦

Reply:
I added it there now